spec/services/weather_man_spec.rb

```diff
@@ -11,20 +11,20 @@
 
   describe ".key_for" do
     it "returns a key for a given zip code and datetime" do
-      key = WeatherMan.key_for(10001, Time.zone.parse('January 1, 2015'))
-      expect(key).to eq "outdoor_temp_for_10001_on_2015-01-01"
+      key = WeatherMan.key_for(Time.zone.parse('January 1, 2015'), 'knyc')
+      expect(key).to eq "outdoor_temp_for_knyc_on_2015-01-01"
     end
 
     it "includes the hour if datetime is today" do
       time = Time.zone.parse('March 1, 2015 at 12am')
-      key = WeatherMan.key_for(10001, time)
-      expect(key).to eq "outdoor_temp_for_10001_on_2015-03-01H00"
+      key = WeatherMan.key_for(time, 'knyc')
+      expect(key).to eq "outdoor_temp_for_knyc_on_2015-03-01H00"
     end
   end
 
   describe ".current_outdoor_temp" do
     it "returns current outdoor temperature from Wunderground API", :vcr do
-      temperature = WeatherMan.current_outdoor_temp(10004, 0)
+      temperature = WeatherMan.current_outdoor_temp('knyc', 0)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.